ts/features/itwallet/components/ItwMarkdown.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { ReactNode } from "react";
```

Comment:
This component could be very useful to other features as well. Can we rename/move the component to make it cross-feature?

Reply:
Yes, it should be like this. At the moment I would leave this component in the scope of itwallet. Considerations of the other streams must be made and some design issues addressed.